app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: It's better to use type hinting for the `cafe` parameter as well. This will make the code more readable and prevent some potential issues. For example: `def go_to_cafe(friends: list, cafe: Cafe) -> str:`